.github/labeler.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+labels:
+- label: "breaking-change"
+  title: "BREAKING CHANGE:.*"
```

Comment:
Should this not be in the footer, in contrast to the title specified here? 
See https://www.conventionalcommits.org/en/v1.0.0/ 

Also, the label is duplicated, would this cause issues?